boring/src/error.rs

```diff
@@ -251,25 +251,11 @@ impl fmt::Debug for Error {
 
 impl fmt::Display for Error {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt, "error:{:08X}", self.code())?;
-        match self.library() {
-            Some(l) => write!(fmt, ":{}", l)?,
-            None => write!(fmt, ":lib({})", ffi::ERR_GET_LIB(self.code()))?,
-        }
-        match self.function() {
-            Some(f) => write!(fmt, ":{}", f)?,
-            None => write!(fmt, ":func({})", ffi::ERR_GET_FUNC(self.code()))?,
-        }
-        match self.reason() {
-            Some(r) => write!(fmt, ":{}", r)?,
-            None => write!(fmt, ":reason({})", ffi::ERR_GET_REASON(self.code()))?,
-        }
         write!(
             fmt,
-            ":{}:{}:{}",
-            self.file(),
-            self.line(),
-            self.data().unwrap_or("")
+            "{} ({})",
+            self.reason().unwrap_or("unknown error"),
+            self.code()
```

Comment:
Would like to keep file/line info. It was very useful previously.

Reply:
They are already printed in the `Debug` impl though.